packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx

```diff
@@ -1,8 +1,11 @@
-import { XLayoutType } from '@lightdash/common';
```

Comment:
Needs this import back so `onXAxisLabelChange` can use it 👍

Reply:
on it